azure-pipelines.yml

```diff
@@ -1,9 +1,11 @@
 pool:
-  name: Hosted VS2017
+  vmImage: 'windows-latest'
 
 variables:
   - name: "Build.ArtifactStagingDirectory"
     value: "Artifacts/"
+  - name: "buildPlatform"
+    value: "x64"

```

Comment:
We don't need this anymore, right?

Reply:
I think not, It was part of the suggestions to fix why it was not building. Now if we remove it and it fails, then we put it back 😄